application/comit_node/src/http_api/mod.rs

```diff
@@ -116,7 +116,8 @@ impl Serialize for Http<SwapProtocol> {
         S: Serializer,
     {
         match &self.0 {
-            SwapProtocol::Rfc003 => serializer.serialize_str("rfc003"),
+            // Currently we do not expose the hash_function protocol parameter via REST.
```

Comment:
We said (dee https://github.com/comit-network/comit-rs/pull/1033/#issuecomment-505192628) that we would return the hash_function in the response on the REST API. Is the comment incorrect or the code and the comment?

Reply:
Ok, comment is now considered invalid.